docs/source/technical_tutorials/user_data/memory-profiling-python.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
<p>"Defining function call" is a bit of an awkward phrase. This also makes it seem like you're calling and profiling only 1 function.</p><p> </p><p>Perhaps something along these lines:</p><p> </p><pre class="ql-syntax" spellcheck="false">@profile
def my_function():
    # ...

@profile
def my_other_function():
    # ...

def main():
    my_function()
    my_other_function()
    # ...

if __name__ == "__main__":
    main()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Ugh! reviewnb totally ruins code blocks!</p>

Reply:
Thanks, @chuckwondo ..Noted with thanks and updated in the recent commit. Requesting re-review